x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/utils.ts

```diff
@@ -95,6 +95,10 @@ export const getTrailingBreadcrumbs = (
     ];
   }
 
+  if (params.state && !params.state.isExistingRule) {
```

Comment:
shouldn't we also have the `isRuleEditPage(params.pathName) && ...` condition here?